src/sqlfluff/dialects/dialect_oracle.py

```diff
@@ -53,7 +54,7 @@
 )
 
 oracle_dialect.sets("unreserved_keywords").update(
-    ["EDITIONABLE", "EDITIONING", "NONEDITIONABLE", "KEEP"]
+    ["EDITIONABLE", "EDITIONING", "NONEDITIONABLE", "KEEP", "+"]
```

Comment:
@joaostorrer - what's the thinking on adding `+` as a keyword here. Normally I would expect that it would parse as a `symbol` rather than a `keyword`.

Is there something I'm missing?

Reply:
That makes more sense.
I just updated the branch with your sugestion.